compiler-rt/lib/gwp_asan/tests/harness.h

```diff
@@ -11,7 +11,7 @@
 
 #include <stdarg.h>
 
-#if defined(__Fuchsia__)
+#ifdef LIBC_COPT_TEST_USE_ZXTEST
```

Comment:
Since this is removing the only occurrence of "Fuchsia" in this file, I'm wondering if we could leave some breadcrumb comment that ties zxtest back to Fuchsia like you did in `libc/test/UnitTest/Test.h`, for the benefit of those who don't know what zxtest is